### DISCUSSION TITLE
Add a multiple-choice input in workflow-dispatch

### DISCUSSION THREAD

**Discussion by HHK1** [community/community#39776] [Actions]
### Select Topic Area

Product Feedback

### Body

The new types for workflow_dispatch inputs are great; 
However I'm trying to use that to work with a matrix strategy, and it's hard to find a good solution.
- I could have some string input and write my array for my matrix strategy directly but that's cumbersome to know what's a valid input
- I've tried to use multiple boolean checks, but that's also not good because of the 10 inputs limit

I'd love to see a new input type to be able to have a multiple select that would output an array of values (strings). That way it could be plugged into a matrix strategy directly.

**Comment by ossanna16** [comment]
Hi there @HHK1 and welcome to our community! Thank you for the feedback!
